app/auxilaries/parsing_shopping.py

```diff
@@ -0,0 +1,99 @@
+import datetime
+import json
+import os
+
+from typing import List
+
+from app.auxilaries.car import Car
+from app.auxilaries.customer import Customer
+from app.auxilaries.shop import Shop
+
+
+def parse_data_from_json() -> list:
+    current_directory = os.path.dirname(__file__)
+    parent_directory = os.path.abspath(
+        os.path.join(current_directory, os.pardir)
+    )
+    config_file_path = os.path.join(parent_directory, "config.json")
+    with open(config_file_path, "r") as json_file:
+        json_data = json.load(json_file)
+
+    json_fuel_price = json_data["FUEL_PRICE"]
+    json_customers = json_data["customers"]
+    json_shops = json_data["shops"]
+
+    return [json_fuel_price, json_customers, json_shops]
+
+
+def create_customer_classes(json_customers: list) -> List[Customer]:
+    customer_list = []
+    for line in json_customers:
+        car = Car(
+            line["car"]["brand"],
+            line["car"]["fuel_consumption"]
+        )
+        customer = Customer(
+            name=line["name"],
+            product_cart=line["product_cart"],
+            location=line["location"],
+            money=line["money"],
+            car=car
+        )
+        customer_list.append(customer)
+
+    return customer_list
+
+
+def create_shops_classes(json_shops: list) -> List[Shop]:
+    shop_list = []
+    for line in json_shops:
+        shop = Shop(
+            name=line["name"],
+            location=line["location"],
+            products=line["products"]
+        )
+        shop_list.append(shop)
+
+    return shop_list
+
+
+def define_destination_shop(
+        customer: Customer,
+        shops: List[Shop]
+) -> Shop:
+    for shop in shops:
+        shop.total_cost = round(
+            (customer.get_fuel_cost(customer.car, shop) * 2
+             + customer.calculate_product_cost(shop)), 2)
```

Comment:
Place the closing parenthesis on a new line to match the opening one and not break the code style.